spec/rails_helper.rb

```diff
@@ -3,9 +3,12 @@
 require 'factory_girl_rails'
 require File.expand_path('../../config/environment', __FILE__)
 require 'rspec/rails'
+require 'support/reservation_helper.rb'
```

Comment:
Same question again - we're sure all three include/require statements are necessary here?